docs: add security policy (#1116)
desmos-labs/desmos: SECURITY.md

# Desmos Security Policy

The Desmos core team and community takes all security issues and vulnerabilities very seriously.

Thanks for improving the security of Desmos. We appreciate your efforts. Following these responsible disclosure guidelines will make sure your contribution is acknowledged.

To report a security issue go to [Discord](https://discord.desmos.network) and alert the core engineers:  

> * Riccardo | Desmo `Riccardo Montagnin#5414`

Reviewer: ```suggestion
* Riccardo | Desmos `Riccardo Montagnin#5414`
```

* Paul | Desmos `Paul | Desmos#0380`
* Manuel | Desmos `Manuel Turetta#8033`

> 

Reviewer: We should also provide an e-mail address. I will add this, and set it up later:
```suggestion
Alternatively, you can also write an email to _security@desmos.network_.

```

_Please avoid opening a public Github issue or posting on social media_.

> The Desmos team will respond with the next steps following the email. The team will keep you informed on the remediation process and may ask for additional guidance/information.

Reviewer: ```suggestion
The Desmos team will respond with the next steps following the report. It will also keep you informed on the remediation process and may ask for additional guidance/information.
```

Please include the following in your report:

- Your name/affiliation (if any)
- Description of the technical details of the vulnerability, including how to reproduce.
- An explanation of who can exploit this vulnerability, including possible attack scenarios.
- Whether this vulnerability is public or known to third parties.

## Vulnerability Disclosure Policy

The core team asks security researchers to keep communications around vulnerabilities private and confidential until a patch is ready.

> Additionally, we request:

Reviewer: ```suggestion
Additionally, we request reporters to:
```

- Allow a reasonable amount of time to correct and address the issue.
- Avoid exploiting the vulnerability.
- Demonstrate good faith by not disrupting Desmos's network, data, or services.

## Vulnerability Disclosure Process

Once a report is received, the following process will be followed:

> - The Desmos core team will work to verify the issue.
> - Work on a patch in a private repository.
> - Notify the community and validators that a security update is coming, giving ample time to upgrade and apply the patch.
> - After the community has been notified, and after verifying that the patch works, the team will pay out any relevant bug bounties to submitters.
> - A post-mortem will be published a week after the vulnerability is discovered.

Reviewer: ```suggestion
1. The Desmos core team will work to verify the issue.
2. The team will work on a patch in a private repository.
3. The team will notify the community and validators that a security update is coming, giving ample time to upgrade and apply the patch.
4. After the community has been notified, and after verifying that the patch works, the team will pay out any relevant bug bounties to submitters.
5. A post-mortem will be published a week after the vulnerability is discovered.
```
